test-variables.yaml.example

```diff
@@ -18,3 +18,8 @@ CRUNCHY:
   privateApiSecret: xxxx
 COCKROACH:
   apiSecretKey: xxxx
+RDS:
+  awsAccessKeyId: xxxx
+  awsSecretAccessKeyId: xxxx
+  awsRegion: xxxx
```

Comment:
These terms are supposed to be concurrent with what RHODA Secrets generates for CLI automation to work correctly. However this is not very critical for UI automation given that we can have only one active RDS provider account per cluster. If we decide to create CLI automation for Amazon RDS, we can align these terms then.

Reply:
OK sounds good to me.